### ISSUE TITLE
"ERROR: Error: Expecting value: line 1 column 1 (char 0)" error when running iotedgehubdev

### ISSUE THREAD

**Issue by BretStateham** [Azure/iotedgehubdev#150]
Hey all, I'm getting the following error when I try to use the iotedgehubdev simulator from VS Code (VS Code has nothing to do with it, same error happens if you just run it from the command line).  The error I'm getting is:
 
```
ERROR: Error: Expecting value: line 1 column 1 (char 0)
```
 
I know I've encountered this before and thought my fix was to (I'm running on Windows 10):

1. Run `pip uninstall iotedgehubdev`   
2. Ensure that the the pip package folders where gone and delete the `C:\Program Data\iotedgehubdev` folder
3. Run `pip install iotedgehubdev`
 
However, this time, that didn't seem to help.  I've verified that it isn't my deployment.json file (created a new deployment manifest from the portal and copied that source in).  
 
Anybody run into this before that can help me out ?

**Comment by BretStateham** [comment]
I should add that I have verified the problem isn't with the deployment.json file I'm trying to use.  I've created a new empty (just edgeAgent and edgeHub modules) manifest in the portal and tested with that.  I get the same error.

**Comment by LazarusX** [comment]
Thanks for reporting this issue.

Could you help share below information?
- The command which caused this error
- The deployment.json which caused this error
- Python version
- iotedgehubdev version

**Comment by LazarusX** [comment]
Hi @BretStateham, we just released iotedgehubdev 0.7.0 which will show more information if something's wrong with the setup  step. Please help upgrade, re-setup, and run. Let us know if it solves your issue.

**Comment by BretStateham** [comment]
@LazarusX my apologies for missing your comments.  I created a new project and ran it from scratch with no issues so there MUST be something wrong with the source files in my initial project although I have yet to be able to figure out what that issue was.  I've just upgraded iotedgehubdev to v.0.7.0 .  I'll give it a try again and see if I can figure out what the issue was. 

Regardless, even though I was adamant that it wasn't my fault it appears that it was.  There must be something broken in my files.
